src/view-backend.c

```diff
@@ -66,14 +66,25 @@ wpe_view_backend_destroy(struct wpe_view_backend* backend)
     backend->input_client = 0;
     backend->input_client_data = 0;
 
+    backend->state = 0;
```

Comment:
This means that the initial state does not have any state bits set. Is it correct to assume that a web view is always initially *not visible*, *not focused* and *not contained in a window*?

Reply:
Perfect, thanks for the clarification.

(Side note, but not for this patch: we should start thinking of adding documentation for the public API.)